packages/default-theme/layouts/default.vue

```diff
@@ -30,33 +30,38 @@ export default {
     SwCart,
     SwFooter,
     SwBottomNavigation,
-    SwPluginTopNavigation
+    SwPluginTopNavigation,
   },
   computed: {
     componentBreadcrumbs() {
       // TODO probably move to vuex now as it's not rendered on server side
       return (
-        this.$route.matched.map(r => {
-          return r.components.default.options.data().breadcrumbs
-        })[0] || {}
+        this.$route.matched
```

Comment:
maybe some kind of helper for such a case?

Reply:
definitely, but not with this issue, here I just covered a case when it sometimes showed an error